Demos/Blazorise.Demo/Pages/Tests/VideoPage.razor

```diff
@@ -19,21 +19,32 @@
                 </Buttons>
                 <Button Color="Color.Primary" Clicked="@OnChangeVideoUrl">Change Video</Button>
                 <Button Color="Color.Primary" Clicked="@OnChangeCurrentTime">Go to 20 seconds</Button>
+                <Button Color="Color.Secondary" Clicked="@OnChangeQuality">Change Quality to 1080</Button>
             </CardBody>
             <CardBody>
-                <Video @ref="@videoRef" Source="@videoUrl" CurrentTime="@currentTime" />
+                <Video @ref="@videoRef" Source="@videoSource" CurrentTime="@currentTime" DefaultQuality="@quality" />
             </CardBody>
         </Card>
     </Column>
 </Row>
 @code {
     Video videoRef;
-    string videoUrl = "http://commondatastorage.googleapis.com/gtv-videos-bucket/sample/BigBuckBunny.mp4";
+    VideoSource videoSource = new VideoSource()
+    {
+        Medias = new ValueEqualityList<VideoMedia>
```

Comment:
What api is this? `ValueEqualityList`. a user is supposed to use it?

Reply:
It's a custom-overloaded list that is used to compare only values instead of the list references.

Reply:
Ok then, I guess it's fine, they'll have to new it up anyway.